lib/send_recover_link.rb

```diff
@@ -0,0 +1,21 @@
+require 'mailgun'
+class SendRecoverLink
+
+ def initialize(mailer: nil)
+   @mailer = mailer || Mailgun::Client.new(ENV["your_api_key"])
+ end
+
+ def self.call(user, mailer = nil)
+   new(mailer: mailer).call(user)
+ end
+
+ def call(user)
+   mailer.send_message(ENV["mailgun_domain_name"], {from: "chitter@mail.com",
+       to: user.email,
+       subject: "reset your password",
+       text: "click here to reset your password http://chitter-ma.com/reset_password?token=#{user.password_token}" })
```

Comment:
Metrics/LineLength: Line is too long. [117/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)